principles.md

```diff
@@ -0,0 +1,34 @@
+# Status Principles
+
+The goal of Status is widespread adoption of the decentralized web, also known as web3. Our challenge is achieving mass adoption while staying true to our principles outlined below.
+
+**Liberty**
+We believe in the sovereignty of individuals. As a platform that stands for the cause of personal liberty, we aim to maximize social, political, and economic freedoms.
```

Comment:
Could we include a complementary statement or section that explains how Status conceives of the platform promoting the [common good](https://en.wikipedia.org/wiki/Common_good), i.e. in relation to or in addition to personal liberty?

Reply:
@michaelsbradleyjr That's a great idea, I think something like this would fit nicely in the annotated version live here: https://hackmd.io/5jB9lWyIQpO8lG1cyLR8oA?both